extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -409,14 +414,15 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
    * @return {!Object<string,string|boolean|number>}
    * @visibleForTesting
    */
-  getPageParameters(consentState) {
+  getPageParameters(consentState, instances = [this]) {
```

Comment:
AMP does not allow default parameters like this. Make it an optional param, and set it within the method like 
`instances = instances || [this]`

Also need to update jsdoc with new param

Reply:
Done.

Are there docs that say what ES5/6/7/n magic we can use in AMP?